// ayu/dist/dark.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var color_1 = require("./color");
var _ = color_1.default('0F1419');
var common = {
    accent: _('F29718'),
    bg: _('0D131A'),
    fg: _('BFBAB0'),
    ui: _('475059')
};
var syntax = {
    tag: _('39BAE6'),
    func: _('FFB454'),
    entity: _('59C2FF'),
    string: _('C2D94C'),
    regexp: _('95E6CB'),
    markup: _('F07178'),
    keyword: _('FF7733'),
    special: _('E6B673'),
    comment: _('626A73'),
    constant: _('FFEE99'),
    operator: _('F29668'),
    error: _('FF3333')
};
var vcs = {
    added: _('91B362'),
    modified: _('6994BF'),
    removed: _('D96C75')
};
var ui = {
    line: common.bg.darken(.13),
    panel: {
        bg: common.bg.brighten(.1),
        shadow: common.bg.darken(.2),
        border: common.bg.darken(.8)
    },
    gutter: {
        normal: common.ui.alpha(.4),
        active: common.ui.alpha(.8)
    },
    selection: {
        bg: vcs.modified.fade(.87),
        inactive: vcs.modified.fade(.92),
        border: vcs.modified.fade(.8)
    },
    guide: {
        active: common.ui.alpha(.7),
        normal: common.ui.alpha(.3)
    }
};
exports.default = {
    common: common,
    syntax: syntax,
    ui: ui,
    vcs: vcs
};
