// ayu/dist/light.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var color_1 = require("./color");
var _ = color_1.default('FAFAFA');
var common = {
    accent: _('F2973D'),
    bg: _('FAFAFA'),
    fg: _('6C7680'),
    ui: _('959DA6')
};
var syntax = {
    tag: _('55B4D4'),
    func: _('F29718'),
    entity: _('399EE6'),
    string: _('86B300'),
    regexp: _('4CBF99'),
    markup: _('F07171'),
    keyword: _('FA6E32'),
    special: _('E6B673'),
    comment: _('ABB0B6'),
    constant: _('A37ACC'),
    operator: _('ED9366'),
    error: _('F51818')
};
var vcs = {
    added: _('99BF4D'),
    modified: _('709ECC'),
    removed: _('F27983')
};
var ui = {
    line: common.ui.alpha(.1),
    panel: {
        bg: common.bg.brighten(.1),
        shadow: common.bg.darken(.3),
        border: common.bg.darken(.2)
    },
    gutter: {
        normal: common.ui.alpha(.4),
        active: common.ui.alpha(.8)
    },
    selection: {
        bg: vcs.modified.fade(.87),
        inactive: vcs.modified.fade(.92),
        border: vcs.modified.fade(.8)
    },
    guide: {
        active: common.ui.alpha(.7),
        normal: common.ui.alpha(.3)
    }
};
exports.default = {
    common: common,
    syntax: syntax,
    ui: ui,
    vcs: vcs
};
